app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
```

Comment:
This change is not really needed IMO. It was fine as it was.

Reply:
As I remember, the `ActionController::API` doesn't have `respond_to` method. But I agree that we should not change this as this is an API application.
Should we just let the `HealthController` inherits `ActionController::Base` only?